src/home.rs

```diff
@@ -243,7 +243,10 @@ where
                     })?;
 
                 if check_res.map(|s| s.trim().to_owned()).as_deref() == Some("true") {
-                    debug!("Using explicit configuration from flag: {}", config_name);
+                    debug!(
+                        "Using explicit configuration from flag/environment: {}",
+                        config_name
+                    );
```

Comment:
```suggestion
                    debug!(
                        "Using explicit configuration from flag/environment: {config_name}",
                        
                    );
```

If I'm not mistaken clippy will tell you this is the correct way. Could also be applied with `sh fix.sh` in the repository root.